tests/integration/render.int.js

```diff
@@ -174,4 +174,46 @@ describe('Full dummy app render', () => {
 					throw err;
 				});
 		}));
+	it('redirects permanently (301) to internal route for <Redirect to={internalPath} permanent />', () =>
+		start(getMockRenderRequestMap(), {}).then(server => {
+			const request = {
+				method: 'get',
+				url: '/redirect/internal/permanent',
+				credentials: 'whatever',
+			};
+			return server
+				.inject(request)
+				.then(response => {
+					expect(response.statusCode).toBe(301);
+					expect(response.headers.location).toBe(
+						INTERNAL_REDIRECT_PATH
+					);
+				})
+				.then(() => server.stop())
+				.catch(err => {
+					server.stop();
+					throw err;
+				});
+		}));
+	it('redirects permanently (301) to external route for <Redirect to={URL object} permanent />', () =>
+		start(getMockRenderRequestMap(), {}).then(server => {
+			const request = {
+				method: 'get',
+				url: '/redirect/external/permanent',
```

Comment:
sorry, being a little 'clever' here in order to use the URLs to conditionally render different kinds of `<MockRedirect>` elements in `mockApp.jsx` - the last two section of the URL path are used to change the props passed to the `<Redirect>` element [here](https://github.com/meetup/meetup-web-platform/pull/300/files/a01145cb76ee165ca2b2f93cd04cc189ddf51347#diff-c22e2430a4ffb0ef9932d7e94b5330f1R20)